src/webextension/manage/components/entryDetails.js

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Localized } from "fluent-react/compat";
+import PropTypes from "prop-types";
+import React from "react";
+
+export default class EntryDetails extends React.Component {
+  static get propTypes() {
+    return {
+      entry: PropTypes.shape({
+        id: PropTypes.number.isRequired,
```

Comment:
{id} as a number fine for now, but will be a string when combined with real data